src/views/login/pronote/PronoteInstanceSelector.tsx

```diff
@@ -299,7 +303,7 @@ const PronoteInstanceSelector: Screen<"PronoteInstanceSelector"> = ({
                       );
                     }}
                     text={instance.name}
-                    subtext={`à ${instance.distance.toFixed(2)}km de toi`}
+                    subtext={prevRoute.name === "PronoteManualLocation" ? "" : `à ${instance.distance.toFixed(2)}km de toi`}
```

Comment:
```suggestion
                    subtext={prevRoute.name === "PronoteManualLocation" ? undefined : `à ${instance.distance.toFixed(2)}km de toi`}
```
Faut juste que tu testes que ça ne fasse pas d'erreur (normalement pas).
C'est pour avoir quelque chose de plus propre, c'est mieux de renvoyer `undefined` que `""`.

Reply:
Pas de soucis